src/sentry/api/endpoints/project_rule_actions.py

```diff
@@ -110,7 +111,10 @@ def execute_future_on_test_event(
                     logger.warning(
                         "%s.test_alert.unexpected_exception", callback_name, exc_info=True
                     )
-                    break
```

Comment:
should we break early so we don't have to stack error messages in the UI?

Reply:
Can do. When I was testing this with multiple integrations, it already breaks on the first error and halts execution, but it wouldn't hurt to do this just in case.